### DISCUSSION TITLE
Prometheus Alertmanager: change alerts path

### DISCUSSION THREAD

**Discussion by nabokihms** [grafana/grafana#42805] [Alerting]
Hello, Grafana team ✋ 

We are researching how to use alerting in Grafana with Prometheus Alertmanager. 
The first one that we are expecting is that it will be possible to configure the `alerts` path of the alertmanager. As I see, not it is hardcoded to `/api/v1/alerts`. https://github.com/grafana/grafana/blob/2af5ad4764c8f2a0c8ab223c5528ba44efde01c3/pkg/services/alerting/notifiers/alertmanager.go#L179

1. Alertmanager has a new endpoint for the alerts - [`/api/v2/alerts`](https://prometheus.io/docs/alerting/latest/clients/), which I believe is preferable to utilize.
2. There is the toggle for switching between APIs in [Prometheus config](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#alertmanager_config) - `api_version`.

In my opinion, we should add an option similar to the one from Prometheus to switch between APIs. What do you think? We are ready to make the change if it is worth it.

**Comment by nabokihms** [comment]
Still an issue for us. Any updates?

**Comment by laa88rf** [comment]
Hello.
It's a perfect idea.
I think the new path must contain ALL alerts with states (Firing/Pending/Normal/etc). Like **/api/v1/alerts**, but better

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
